src/contract.py

```diff
@@ -105,16 +112,10 @@ def _make_invoke_function(self, *args, signature=None, **kwargs):
         return InvokeFunction(
             contract_address=self.contract_data.address,
             entry_point_selector=self.selector,
-            calldata=self._make_calldata(*args, **kwargs),
+            calldata=self._transformer(*args, **kwargs),
```

Comment:
i kind of dislike the way of calling this transformer, can we just call `from_python` ?